
# Topic type (e.g. 'std_msgs/String')
string type

# Topic md5 sum
uint32[4] md5

uint8[] data
